crates/hyperswitch_connectors/src/connectors/taxjar.rs

```diff
@@ -194,7 +194,18 @@ impl ConnectorIntegration<CalculateTax, PaymentsTaxCalculationData, TaxCalculati
             req.request.currency,
         )?;
 
-        let connector_router_data = taxjar::TaxjarRouterData::from((amount, shipping, req));
+        let order_amount = utils::convert_amount(
+            self.amount_converter,
+            req.request
+                .order_details
+                .as_ref()
+                .and_then(|details| details.first().map(|item| item.amount))
+                .unwrap_or(MinorUnit::new(0)),
```

Comment:
I think there is `MinorUnit::Zero()` for these cases, please try avoiding using `MinorUnit::new()` in connector files